libmscore/layout.cpp

```diff
@@ -3123,17 +3123,20 @@ void Score::layoutLyrics(System* system)
 
 void layoutTies(Chord* ch, System* system, int stick)
       {
+      SysStaff *staff = system->staff(ch->staffIdx());
+      if(! staff->show())
```

Comment:
Pedantic style comment: why the space after the "!" ? Common style throughout the rest of the codebase doesn't have the space.